src/Aspire.Dashboard/Model/ValidateTokenMiddleware.cs

```diff
@@ -88,7 +88,10 @@ public static async Task<bool> TryAuthenticateAsync(string incomingBrowserToken,
             authenticationType: CookieAuthenticationDefaults.AuthenticationScheme);
         var claims = new ClaimsPrincipal(claimsIdentity);
 
-        await httpContext.SignInAsync(CookieAuthenticationDefaults.AuthenticationScheme, claims).ConfigureAwait(false);
+        await httpContext.SignInAsync(
+            CookieAuthenticationDefaults.AuthenticationScheme,
+            claims,
+            new AuthenticationProperties { IsPersistent = true, AllowRefresh = true }).ConfigureAwait(false);
```

Comment:
Nit: `AllowRefresh = true` is the [default behavior](https://github.com/dotnet/aspnetcore/blob/7412c71cf881392233b5652310ab2b9e7fdf71eb/src/Security/Authentication/Cookies/src/CookieAuthenticationHandler.cs#L97) if you want to rely on that. I doubt we'd change it.

```suggestion
            new AuthenticationProperties { IsPersistent = true }).ConfigureAwait(false);
```